tests/unit/util/test_tag_yaml_loader.py

```diff
@@ -0,0 +1,412 @@
+# pylint: disable=missing-docstring
+# pylint: disable=no-self-use
+# pylint: disable=line-too-long
+# pylint: disable=unspecified-encoding
+# pylint: disable=protected-access
+import tempfile
+
+import pytest
+from ruamel.yaml import YAML
+from ruamel.yaml.constructor import ConstructorError
+
+from logprep.util.tag_yaml_loader import init_yaml_loader_tags
+
+
+@pytest.fixture(name="yaml_directory")
+def fixture_yaml_directory() -> str:
+    return tempfile.mkdtemp()
```

Comment:
consider to use the `tmp_path` fixture.
see: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html